Qualisys/QTMConnectLiveLink/Source/QTMConnectLiveLink/Private/QTMConnectLiveLinkSource.cpp

```diff
@@ -300,7 +317,8 @@ uint32 FQTMConnectLiveLinkSource::Run()
             {
                 components |= CRTProtocol::cComponentSkeleton;
             }
-            if (!mRTProtocol->StreamFrames(CRTProtocol::RateAllFrames, 0, 0, nullptr, components))
+            CRTProtocol::EStreamRate streamRate = (Settings.StreamRate == "Frequency") ? CRTProtocol::RateFrequency : (Settings.StreamRate == "Frequency Divisor") ? CRTProtocol::RateFrequencyDivisor : CRTProtocol::RateAllFrames;
```

Comment:
I think this might as well be if/else if instead. Would be more readable.

Reply:
Fixed in commit [`b7c49eb`](https://github.com/qualisys/QTM-Connect-For-Unreal/pull/13/commits/b7c49ebdd5ec2a362286ea2ce7437be7f53d9ba3)